web/src/components/FilingsCalendar.test.tsx

```diff
@@ -476,6 +476,51 @@ describe("<FilingsCalendar />", () => {
         screen.queryByText(Config.dashboardDefaults.calendarGridViewButton, { exact: false })
       ).not.toBeInTheDocument();
     });
+
+    it("doesn't display past months in list view", () => {
+      const pastDueDate = getCurrentDate().subtract(1, "months");
+      const pastReport = generateTaxFiling({
+        identifier: "past",
+        dueDate: pastDueDate.format(defaultDateFormat),
+      });
+
+      const futureDueDate = getCurrentDate().add(2, "months");
```

Comment:
This works, and knowing what we know about upcoming stories, I don't think it needs to be changed. That said, I just want to point out that this is a dangerous test because if it was November or December, this test would suddenly start failing out of nowhere. Because our current logic is to only show events in the future but still in the calendar year. If the current date is November, then 2 months out is January - a different year, so it wouldn't show up. 

I wanted to point this out so that you have it in your mind in general when writing date tests. Again, doesn't need to be changed since we know this behavior is changing so soon. But if I were to change this, I would probably explicitly pick a date I know to be in the future but in this year: `futureDueDate = dayjs(new Date(getCurrentDate.year(), 12, 31))`